frontend/src/components/UrlNotFound.tsx

```diff
@@ -0,0 +1,26 @@
+import {Box, Typography} from "@mui/material";
+import React from "react";
+import {BsSignStopFill} from "react-icons/bs";
+
+export default function UrlNotFound() {
+    return (
+        <>
+            <Box
+                sx={{
+                    display: 'flex',
+                    flexWrap: 'wrap',
+                    '& > :not(style)': {
+                        m: 2,
+                        width: "100%",
+                    },
+                    mt: 20
+                }}
+            >
```

Comment:
Weißt du bestimmt, aber wenn du nicht alles Styling inline willst, kannst du es in einer const auslagern: 
`const sxStyle = {}`